float32 x
float32 y
float32 theta
bool camera_control
---
string response